bin/glm

```diff
@@ -0,0 +1,253 @@
+        printMessage(`Labels synchronized!
+      ${summary.created} created
+      ${summary.deleted} deleted
+      ${summary.updated} updated
+        `);
+      }
+    } else if (command === 'init') {
+      await writeLabelsConfig();
+      printMessage('Labels successfully stored!');
+    } else {
+      printHelp();
+    }
+  } catch (err) {
+    console.error(err);
+    printHelp();
+    process.exit(1);
+  }
+})();
+
+function writeLabelsConfig() {
+  return new Promise(async (resolve, reject) => {
+    try {
+      const labelData = await fetchLabels();
+      const labels = await labelFormatter(labelData);
+      fs.writeFile('./label-config.json', JSON.stringify(labels, null, 2));
+      resolve();
+    } catch (err) {
+      reject(err);
+    }
+  });
+}
+
+function labelControl() {
+  return new Promise(async (resolve, reject) => {
+    try {
+      const labelData = await fetchLabels();
+      const formattedPublishedLabels = await labelFormatter(labelData);
+      fs.readFile('./label-config.json', async (err, data) => {
+        if (err) {
+          throw err;
+        }
+        const localConfig = JSON.parse(data.toString());
+        const newLabels = await checkForNewOrOldLabels(
+          localConfig,
+          formattedPublishedLabels
+        );
+        const labelsToDelete = await checkForNewOrOldLabels(
+          formattedPublishedLabels,
+          localConfig
+        );
+        const labelsToUpdate = await checkForModifications(
+          localConfig,
+          formattedPublishedLabels
+        );
+        if (newLabels.length !== 0) {
+          await createLabels(newLabels);
+        }
+        if (labelsToDelete.length !== 0) {
+          await deleteLabels(labelsToDelete);
+        }
+        if (labelsToUpdate.length !== 0) {
+          await updateLabels(labelsToUpdate);
+        }
+        const changes =
+          newLabels.length + labelsToDelete.length + labelsToUpdate.length;
+        const summary =
+          changes !== 0
+            ? {
+                created: newLabels.length,
+                deleted: labelsToDelete.length,
+                updated: labelsToUpdate.length,
+              }
+            : 'Labels already up to date!';
+        resolve(summary);
+      });
+    } catch (err) {
+      reject(err);
+    }
+  });
+}
+
+function checkForNewOrOldLabels(newLabels, oldLabels) {
+  return new Promise((resolve, reject) => {
+    try {
+      const updatedLabels = [];
+      const OldLabelIds = [];
+      oldLabels.forEach(label => {
+        OldLabelIds.push(label.id ? label.id : undefined);
+      });
+      newLabels.forEach(label => {
+        if (!OldLabelIds.includes(label.id ? label.id : undefined)) {
+          updatedLabels.push(label);
+        }
+      });
+      resolve(updatedLabels);
+    } catch (err) {
+      reject(err);
+    }
+  });
+}
+
+function checkForModifications(localConfig, publishedConfig) {
+  return new Promise(async (resolve, reject) => {
```

Comment:
This is not necessary, you should just make `checkForModifications` async

Reply:
👍  567f587